### ISSUE TITLE
Alt text

### ISSUE THREAD

**Issue by kwahl** [kwahl/AccessibilityTesting#1]
Alt text is missing for a number of images on the page. I'd like to identify the images that we need to have descriptive alt text for and those that need "null" text.
